docs/output.md

```diff
@@ -2,42 +2,110 @@
 
-![MultiQC - FastQC adapter content plot](images/mqc_fastqc_adapter.png)
+### Higher-level-Processing
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `higher-level/<TILE>/`
+  - `param_files/`: Parameter files used in [force-higher-level](https://force-eo.readthedocs.io/en/latest/components/higher-level/index.html).
+  - `trend_files`: Symlinks to trend files that are the result of higher-level processing.
+
+</details>
+
+Higher level processing consist of two parts, generating parameter files and performing various processing task as defined in the parameter files.
+
+Parameter files may be consulted to derive information about the specific processing task performed for a given tile. In this workflow, classification using spectral unmixing is performed.
+
+Spectral unmixing is a common technique to derive sub-pixel classification. Concretely, a set of endmember (provided using `--endmember`) is consulted to determine fractions of different types of vegetation, soil, $\ldots$ for each pixel.
+
+Next, time series analysis for different vegetation characteristics is performed.
+
+The resulting trend files can be consulted to view trends for individual tiles. They are saved as symlinks because of their large size.
+
+If the `return_tss` parameter was set to `true`, the pipeline will also output files with the `TSS` ending. These files contain the time series stack(TSS) for the given tile and index or band. Here, for each date of acquisition, an image is available that contains the values for that date.
+
+### Visualization
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `trend/`
+  - `mosaic/<PRODUCT>/`
+    - `<TILE>`: Auxiliary files for the mosaic visualization.
+    - `mosaic`: Contains a single virtual raster file that defines the mosaic visualization.
+  - `pyramid/<TREND_TYPE>/trend/<TILE>/`: Contains tile-wise pyramid visualizations for every trend analyzed in the workflow.
+
+</details>
 
-:::note
-The FastQC plots displayed in the MultiQC report shows _untrimmed_ reads. They may contain adapter sequence and potentially regions with low quality.
-:::
+Two types of common visualizations are generated in the last step of the pipeline. They are results of [force-mosaic](https://force-eo.readthedocs.io/en/latest/components/auxilliary/mosaic.html) and [force-pyramid](https://force-eo.readthedocs.io/en/latest/components/auxilliary/pyramid.html). Note that these visualizations do not add more logic to the workflow but rather rearrange the output files of higher-level-processing. Both visualizations are enabled by default but may be disabled in a certain configuration files. Thus, these outputs are optional.
 
 ### MultiQC
```

Comment:
remove the mention of FastQC in the tool description

Reply:
Which mention are you referring to?

Reply:
line 124